methods/array/dropWhile.js

```diff
@@ -0,0 +1,27 @@
+function dropWhile(array, identity) {
+  if (!Array.isArray(array)) {
+    throw new TypeError('First argument must be an array');
+  }
+  let part_array = [];
+  let index = 0;
+  for (let i = 0; i < array.length; i++) {
+    if (typeof identity === 'function' && !identity(array[i])) {
+      part_array[index++] = array[i];
+    }
+    if (identity instanceof Array) {
+      if (!JSON.stringify(array[i]).includes(identity[0]) ||
```

Comment:
wow, this is quite tricky. I understand (probably) what's going on here, but I'd like to recommend do it using just a pure javascript. 
In general, I like this, it's smart :)